libseam_carving/tests/data.json

```diff
@@ -1098,7 +1098,7 @@
                     94,
                     76,
                     129,
-                    66,
+                    112,
```

Comment:
Again, incorrect in the spreadsheet and quickly updated here. The problem was that the seam was incorrect, thus the resulting matrix from removing it would also be incorrect.